tigerpath/majors_and_certificates/majors/AST.json

```diff
@@ -160,15 +160,13 @@
             "AST 309/MAE 309/PHY 309/ENE 309: The Science of Nuclear Energy: Fission and Fusion",
             "AST 401: Cosmology",
             "AST 403: Stars and Star Formation",
-            "COS 126: General Computer Science",
-            "MAE 222: Mechanics of Fluids",

```

Comment:
https://web.astro.princeton.edu/academic/undergraduate-program/major-requirements

"Recommended Courses in addition to the above:

Computer Science 126:   General Computer Science
Math 309/ORF 309:      Probability and Stochastic Systems
Mechanical and Aerospace Eng. 222:   Mechanics of Fluids"

Since they are only recommended, I removed these courses as possible courses to fulfill the req.

Reply:
Great catch! 🙌